### PR TITLE
docs: introduce typegraph_std

### PR DESCRIPTION
**Pull request by afmika**
This solves MET-169

### CODE REVIEW

File: website/docs/tutorials/import-your-existing-apis/index.mdx [hunk 1]
```diff
@@ -56,3 +56,15 @@ Effects are a property of materializers and help categorization what happens to
 | `delete` | DELETE     | `DELETE`             |
 
 They provide hints to the typegates for the query orchestration by splitting the queries and mutations. For example, the `create` effect is exposed as a mutation. They also allow setting different policies based on them, that's for the next page.
+
+# Introduction to typegraph_std
```
Comment (zifeo) [inline website/docs/tutorials/import-your-existing-apis/index.mdx:60]: Can you make it a second level title with `##`? html good practice advise for a single top-level title

File: website/docs/tutorials/import-your-existing-apis/index.mdx [hunk 2]
```diff
@@ -56,3 +56,15 @@ Effects are a property of materializers and help categorization what happens to
...
+
+Alternatively, you can use the typegraph_std package for the most common APIs.
+
+It currently includes:
+- Google API (fcm, firebase, youtube, etc.)
+- Stripe
+- Github
+
+They work pretty much the same way as Importers.
```
Comment (zifeo) [inline website/docs/tutorials/import-your-existing-apis/index.mdx:69]: little **I**mp?

File: website/docs/tutorials/import-your-existing-apis/typegraph_std.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+with TypeGraph(
+    "emails",
+    # skip:next-line
+    cors=TypeGraph.Cors(allow_origin=["https://metatype.dev", "http://localhost:3000"]),
+) as g:
+    public = policies.public()
+
+    # {protocol}://{hostname}/api/v3
+    github = import_ghes({"protocol": "https", "hostname": "custom_host"})
```
Comment (zifeo) [inline website/docs/tutorials/import-your-existing-apis/typegraph_std.py:13]: can we have a default setup for import_ghes? can we also use import_github so it is maybe clearer to someone not familiar with the package?

**Review by zifeo — CHANGES_REQUESTED** [review]
Feel free to also update the fcm example above - it is likely better for the user to have a dedicated how-to guide on using the importer and have the tutorial directly using it (when learning, we have to keep it simple).

File: website/docs/tutorials/import-your-existing-apis/index.mdx [hunk 4]
```diff
@@ -56,3 +56,15 @@ Effects are a property of materializers and help categorization what happens to
 | `delete` | DELETE     | `DELETE`             |
 
 They provide hints to the typegates for the query orchestration by splitting the queries and mutations. For example, the `create` effect is exposed as a mutation. They also allow setting different policies based on them, that's for the next page.
+
+# Introduction to typegraph_std
+
+Alternatively, you can use the typegraph_std package for the most common APIs.
+
+It currently includes:
+- Google API (fcm, firebase, youtube, etc.)
```
Comment (Natoandro) [inline website/docs/tutorials/import-your-existing-apis/index.mdx:65]: APIs ?

File: website/docs/tutorials/import-your-existing-apis/typegraph_std.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
+from typegraph import TypeGraph, policies
+from typegraph_std.github.ghes import import_ghes
+from typegraph_std.google.gmail import import_gmail
```
Comment (Natoandro) [inline website/docs/tutorials/import-your-existing-apis/typegraph_std.py:3]: I suggest to enable simpler imports: `from typegraph_std.google import import_gmail`

**Review by zifeo — APPROVED** [review]

**Pull request merged**